src/design-patterns/MazeGame/bombed_maze_factory.cpp

```diff
@@ -0,0 +1,26 @@
+#include "bombed_maze_factory.hpp"
+
+#include <iostream>
+
+#include "bombed_wall.hpp"
+#include "room_with_a_bomb.hpp"
+
+BombedMazeFactory::BombedMazeFactory() {
+  std::cout << "Init BombedMazeFactory()" << std::endl;
+}
+
+std::unique_ptr<Maze> BombedMazeFactory::MakeMaze() const {
+  return std::make_unique<Maze>();
+}
+
+std::unique_ptr<Wall> BombedMazeFactory::MakeWall() const {
+  return std::make_unique<BombedWall>();
+}
+
+std::unique_ptr<Room> BombedMazeFactory::MakeRoom(int n) const {
```

Comment:
```suggestion
std::unique_ptr<Room> BombedMazeFactory::MakeRoom(const int n) const {
```
nit.
의미 있는 매개변수 이름이면 좋겠지만, 뭐; 
변경되지 않는 값을 받아올 땐 실수를 줄이기 위해서라도 상수값 받아오도록 하는게 좋음